core/core-spacemacs-buffer.el

```diff
@@ -811,18 +811,22 @@ border."
     (force-mode-line-update)
     (spacemacs-buffer/goto-link-line)))
 
-(defun spacemacs-buffer/goto-buffer ()
+(defun spacemacs-buffer/goto-buffer (&optional refresh)
   "Create the special buffer for `spacemacs-buffer-mode' if it doesn't
 already exist, and switch to it."
   (interactive)
-  (unless (buffer-live-p (get-buffer spacemacs-buffer-name))
+  (when (or refresh
+              (not  (buffer-live-p (get-buffer spacemacs-buffer-name))))
     ;; revise banner length in GUI
     (when (display-graphic-p)
       (setq spacemacs-buffer--banner-length
-            (- (window-total-width) 2)))
+            (window-width)))
     (with-current-buffer (get-buffer-create spacemacs-buffer-name)
       (page-break-lines-mode)
       (save-excursion
+        (when refresh
+          (let ((buffer-read-only nil))
```

Comment:
`inhibit-read-only` also ignores read-only text properties (see the variable's documentation)